understanding/20/pause-stop-hide.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Pause, Stop, Hide</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Fewer users are distracted by content that updates or moves</dd>
-		<dt>Author task</dt><dd>Let users control content changes that occur in parallel with other content</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Fewer users are distracted by content that updates or moves.</dd>
+         <dt>What to do</dt><dd>Let users control content changes that occur in parallel with other content.</dd>
+         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.</dd> 
```

Comment:
@rainbreaw suggested 

```suggestion
         <dt>Why it's important</dt><dd>Continual movement can be detrimental for some individuals with cognitive disabilities, attention barriers, and vestibular or other sensory processing disorders.</dd> 
```

Reply:
I'd appreciate this level of detail and exact information in the long version within the understanding document. It certainly is good to know.

Yet I am afraid it is too long for the in brief section, and the wording is not easy to understand.

Reply:
@GreggVan like this recommended change.

Reply:
@rainbreaw, in regards to highlighting sensory processing disorders and vestibular considerations, as with several other criteria previously discussed, we would be introducing a concept not covered in the Understanding document.
It's another situation where a PR to update that information needs to take place before considering whether its necessary/desirable to modify the Why text. 
In this particular case, we need to be especially careful with scope and cause and effect. As far as I know, we do not have any data suggesting a ticker tape would trigger episodes, etc.